robotbase/src/main/java/frc/robot/commands/intake/IntakeNoteCommand.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.commands.intake;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Constants;
+import frc.robot.Robot;
+
+/*
+Command needs to:
+
+1. Run intake at set speed (unknown constant)
+
+2. Complete and stop the intake rollers when
+beam-break sensor's beam is broken
+*/
+public class IntakeNoteCommand extends Command {
```

Comment:
We do not append the word `Command` to out commands. We have decided that doing this is unnecessary lego naming.